### ISSUE TITLE
Premium without gplay?

### ISSUE THREAD

**Issue by mannp** [AChep/keyguard-app#29]
Hi, looking to support with lifetime premium but have devices without gplay.

Is there a way to get one time premium standalone?

Thanks.

**Comment by AChep** [comment]
Hi,

At this moment the Play store is the only way. 😔

**Comment by mannp** [comment]
Thanks for the response, hopefully its something you consider for the future 🙂

With sync requiring premium from what I can see, its makes it unusable really for me on none gplay devices; which is a shame.

Thanks for creating though, it looks slick, even after just a quick test.

**Comment by txyyh** [comment]
Now you can buy this app in a device with gplay, then you can direct download premium apk without google server.

But until now those apps are too old, remain in version 0.2

**Comment by AChep** [comment]
> Now you can buy this app in a device with gplay, then you can direct download premium apk without requiring google service.
> 
> But until now those apps are too old, remain in version 0.2

That's correct. The reason is that that type of build got broken after going multiplatform. I plan to revise that soon, maybe I will automate that better or maybe reach time a version is built it will also build an artifact that will be available on GitHub...

**Comment by fahadalisarwar1** [comment]
> Any updates on this? I bought the premium version (one-time payment) because I need the multiple accounts feature. Now I'd like to switch to a de-googled phone. Any way I can transfer my premium license?

you can downlaod apk

**Comment by TridentGalaxy** [comment]
What do you mean, I can download the apk? Does the apk have all the premium features unlocked without the need for payment?

**Comment by eightHundreds** [comment]
Google Play's paid Pro version and the version I compile myself from the GitHub source code cannot be guaranteed to be exactly the same (due to Google App Signing, optimizations, etc.). This makes me a bit worried whether the paid Play Store version might differ from the public source code.
Do you have any plans to implement Reproducible Builds, or other ways that allow users to verify that the Play Store version matches the open source code?

**Comment by AChep** [comment]
> Google Play's paid Pro version and the version I compile myself from the GitHub source code cannot be guaranteed to be exactly the same (due to Google App Signing, optimizations, etc.). This makes me a bit worried whether the paid Play Store version might differ from the public source code.
> Do you have any plans to implement Reproducible Builds, or other ways that allow users to verify that the Play Store version matches the open source code?

Not really, I do not think there's anything that allows you to confirm that the Play store build is built exactly from this code.
